teuthology/openstack/__init__.py

```diff
@@ -756,8 +782,13 @@ def run_suite(self):
                 " ~/.teuthology.yaml"
             ).format(opt='ceph_git_url', value=ceph_repo)
             self.ssh(command)
-        argv.append('/home/' + self.username +
-                    '/teuthology/teuthology/openstack/openstack.yaml')
+        teuthology_repo = '/home/' + self.username + '/teuthology'
+        if self.args.test_repo:
+            argv.append(teuthology_repo +
+                        '/teuthology/openstack/openstack-basic.yaml')
+        else:
+            argv.append(teuthology_repo +
+                        '/teuthology/openstack/openstack.yaml')
```

Comment:
please:
1. change this line to `openstack-buildpackages.yaml`
2. rename the file `openstack.yaml` to `openstack-buildpackages.yaml`
3. remove the duplicated code from `openstack-buildpackages.yaml`

Reply:
(updated the previous comment) so here we would have:

```
+        else:
+            argv.append(teuthology_repo +
+                        '/teuthology/openstack/openstack-basic.yaml')
+            argv.append(teuthology_repo +
+                        '/teuthology/openstack/openstack-buildpackages.yaml')